thrust/benchmarks/bench/copy/basic.cu

```diff
@@ -52,7 +52,40 @@ static void basic(nvbench::state& state, nvbench::type_list<T>)
   });
 }
 
-using types = nvbench::type_list<nvbench::uint8_t, nvbench::uint16_t, nvbench::uint32_t, nvbench::uint64_t>;
+// Non-trivially-copyable/relocatable type which cannot be copied using std::memcpy or cudaMemcpy
+struct NonTrivial
```

Comment:
suggestion: we use pascal case only in template parameters

Reply:
Yeah right. Thx!